src/org/hitachivantara/ci/build/impl/MavenBuilder.groovy

```diff
@@ -302,6 +304,8 @@ class MavenBuilder extends AbstractBuilder implements IBuilder, Serializable {
           "DOCKER_PULL_HOST=${buildData.getString(DOCKER_RESOLVE_REPO)}",
           "DOCKER_PUBLIC_PUSH_HOST=${buildData.getString(DOCKER_PUBLIC_PUSH_REPO)}",
           "DOCKER_PRIVATE_PUSH_HOST=${buildData.getString(DOCKER_PRIVATE_PUSH_REPO)}",
+          "nodeDownloadRoot=${buildData.getString(NODEJS_BUNDLE_REPO_URL)}",
```

Comment:
Don't think this is going to work. This is to create `env` variables not local groovy variables. Is there any build examples that hows this working?

Reply:
yes created the local variable and its resolving to actual values in maven option for XUL snapshot job. Testing the build with Steve.